assets/js/index.js

```diff
@@ -0,0 +1,249 @@
+const API_KEY = "b248c5c1ee789732d38aa9fe65ef7935";
+const form = document.getElementById("form");
+const currentWeatherContainer = document.getElementById("currentWeather");
+const forecastWeatherContainer = document.getElementById("forecastWeather");
+let city = "";
+let lat = "";
+let lon = "";
+let temp = "";
+let icon = "";
+let wind = "";
+let humidity = "";
+let description = "";
+let tempDate = "";
+let uv = "";
+const renderRecentCities = () => {
+  const container = document.getElementById("recent");
+  const searchContainer = document.getElementById("list");
+  searchContainer.remove();
+  $("#recent").append(` <div id="list" class="list-group">
+            
+  </div>`);
+  // get recent cities from LS []
+  const recentCities = JSON.parse(localStorage.getItem("recentSearches"));
+  // if [] is empty then render alert
+  if (recentCities !== null) {
+    for (i = 0; i < recentCities.length; i++) {
+      let city = recentCities[i];
+      $("#list")
```

Comment:
You have another set of duplicated code here - you declared the `searchContainer` on line 17 which is identical to the `$("#list")` you have here. This is a symptom of you not consistently using jQuery syntax throughout.